src/main/java/core/basesyntax/db/Storage.java

```
@@ -0,0 +1,21 @@
+package core.basesyntax.db;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruitsStorage = new HashMap<>();
+
+    public static void putFruitQuantity(String fruit, int quantity) {
+        fruitsStorage.put(fruit, quantity);
+    }
+
+    public static int getFruitQuantity(String fruit) {
+        return fruitsStorage.getOrDefault(fruit, 0);
```

`getFruitQuantity` does not validate its parameter. The Storage/DAO requirements state inputs must be validated: fruit must be non-null and non-blank. Add validation and throw an informative RuntimeException when the fruit key is invalid instead of silently returning a default for invalid input.